Automattic/themes
# Update Node version in .nvmrc & update deps

#### Changes proposed in this Pull Request:
This PR updates the minimum required Node version from 17 to 18, as I was seeing the following error in this repo:

<img width="592" alt="Screenshot 2023-05-05 at 12 36 31" src="https://user-images.githubusercontent.com/1645628/236448666-edfee4ec-26d6-473f-8d71-d88eebcf94c9.png">

This also updates all dependencies.

Testing instructions:

1. Run `rm -rf node_modules` to delete your current node_modules
2. Run `nvm use` to switch to node v18
3. Run `npm ci` to install the updated modules from the package-lock file in this PR
4. There should be no changes to the package-lock file once you've run all these commands
